src/main/java/seedu/address/model/module/Module.java

```diff
@@ -154,6 +174,16 @@ public ZoomLink getLink() {
         return this.zoomLink;
     }
 
+    /**
+     * Returns the zoom link based on the specified key.
+     *
+     * @param key the specified key
+     * @return zoom link
+     */
+    public ZoomLink getLink(String key) {
```

Comment:
Maybe we should perform checks on whether the key exists in the hashmap?

Reply:
Yes we can add a check here, but I think the `HashMap#get()` will already return a null if the key does not exist? 

In addition, I just realized that this method will only be used for editing zoomlinks? Since we will not be implementing a command to query a specific zoomlink. Because of this, I will add a getter method for the entire zoomLink i.e. `getAllZoomLinks`.

Reply:
Yes, that sounds good.